lib/errors.js

```diff
@@ -38,6 +38,10 @@ exports.ForceDisconnectError = function (message) {
   this.message = message;
 };
 
+exports.ForceMuteError = function (message) {
```

Comment:
you added it in the middle of the ForceDisconnectError - move it under it, and connect the new error to the prototype like the other examples